### PR TITLE
Add container_blkio_device_usage metric

### PR DESCRIPTION
**Pull request by kwisniewski98**
Signed-off-by: Wisniewski, Krzysztof2 <Krzysztof2.Wisniewski@intel.com>

### CODE REVIEW

File: metrics/testdata/prometheus_metrics [hunk 1]
```diff
@@ -87,6 +95,9 @@ container_fs_limit_bytes{container_env_foo_env="prod",container_label_foo_label=
 # TYPE container_fs_read_seconds_total counter
 container_fs_read_seconds_total{container_env_foo_env="prod",container_label_foo_label="bar",device="sda1",id="testcontainer",image="test",name="testcontaineralias",zone_name="hello"} 2.7e-08 1395066363000
 container_fs_read_seconds_total{container_env_foo_env="prod",container_label_foo_label="bar",device="sda2",id="testcontainer",image="test",name="testcontaineralias",zone_name="hello"} 4.2e-08 1395066363000
+# HELP container_fs_reads_bytes_total Cumulative count of bytes read
```
Comment (katarzyna-z) [inline metrics/testdata/prometheus_metrics:98]: It is strange that you've needed to add this metrics to test. There are not in scope of this pull request, please investigate it.

File: docs/storage/prometheus.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ Metric name | Type | Description | Unit (where applicable) | -disable_metrics pa
 `container_accelerator_duty_cycle` | Gauge | Percent of time over the past sample period during which the accelerator was actively processing | percentage | accelerator |
 `container_accelerator_memory_total_bytes` | Gauge | Total accelerator memory | bytes | accelerator |
 `container_accelerator_memory_used_bytes` | Gauge | Total accelerator memory allocated | bytes | accelerator |
+`container_blkio_device_usage` | Counter | Blkio device bytes usage | | diskIO |
```
Comment (Creatone) [inline docs/storage/prometheus.md]: Add bytes to the unit column.

File: metrics/prometheus.go [hunk 3]
```diff
@@ -757,6 +757,28 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
...
 			},
+			{
+				name:        "container_blkio_device_usage",
+				help:        "Blkio Device bytes usage",
+				valueType:   prometheus.CounterValue,
+				extraLabels: []string{"device", "major", "minor", "operation"},
+				getValues: func(s *info.ContainerStats) metricValues {
+					var values metricValues
+					for _, diskStat := range s.DiskIo.IoServiceBytes {
+						for key, value := range diskStat.Stats {
```
Comment (Creatone) [inline metrics/prometheus.go]: If you use the key as operation why not name it "operation"?

File: metrics/testdata/prometheus_metrics (hunk 1 above)
Comment (kwisniewski98) [reply metrics/testdata/prometheus_metrics:98]: By adding `DiskIo` to `prometheus_fake.go` ( which is needed to test new metric) `container_fs_reads_bytes_total` metric was created. Seems like someone who added that metric didn't test it.

File: metrics/prometheus_fake.go [hunk 4]
```diff
@@ -524,6 +524,21 @@ func (p testSubcontainersInfoProvider) GetRequestedContainersInfo(string, v2.Req
...
 						},
 					},
+					DiskIo: info.DiskIoStats{
+						IoServiceBytes: []info.PerDiskStats{{
+							Device: "/dev/sdb",
+							Major:  8,
+							Minor:  0,
+							Stats: map[string]uint64{
+								"Async":   0,
+								"Discard": 0,
```
Comment (katarzyna-z) [inline metrics/prometheus_fake.go]: Usually I set different values for different keys in tests to be sure that value from different keys are read correctly.

File: metrics/testdata/prometheus_metrics (hunk 1 above)
Comment (katarzyna-z) [reply metrics/testdata/prometheus_metrics:98]: If tests fail before your changes please open another pull request with appropriate fix.

File: docs/storage/prometheus.md [hunk 5]
```diff
@@ -21,6 +21,7 @@ Metric name | Type | Description | Unit (where applicable) | -disable_metrics pa
 `container_accelerator_duty_cycle` | Gauge | Percent of time over the past sample period during which the accelerator was actively processing | percentage | accelerator |
 `container_accelerator_memory_total_bytes` | Gauge | Total accelerator memory | bytes | accelerator |
 `container_accelerator_memory_used_bytes` | Gauge | Total accelerator memory allocated | bytes | accelerator |
+`container_blkio_device_usage` | Counter | Blkio device bytes usage | bytes | diskIO |
```
Comment (Creatone) [inline docs/storage/prometheus.md]: Please change it to `container_blkio_device_usage_bytes_total`

File: metrics/prometheus.go [hunk 6]
```diff
@@ -757,6 +757,28 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					}, s.Timestamp)
 				},
 			},
+			{
+				name:        "container_blkio_device_usage",
```
Comment (Creatone) [inline metrics/prometheus.go]: Please change it to `container_blkio_device_usage_bytes_total`

**Pull request closed without merging**